crates/swc_ecma_minifier/tests/benches-full/vue.js

```diff
@@ -185,7 +185,7 @@
     UA && UA.indexOf('android');
     var isIOS = UA && /iphone|ipad|ipod|ios/.test(UA) || 'ios' === weexPlatform;
     UA && /chrome\/\d+/.test(UA), UA && /phantomjs/.test(UA);
-    var isFF = UA && UA.match(/firefox\/(\d+)/), nativeWatch = {}.watch, supportsPassive = !1;
```

Comment:
This looks like a bug

Reply:
`{}.watch` is replaced with `void 0` since the property `watch` doesn't exist on `Object`'s prototype, but I think I'll add `watch` and `unwatch` anyway to avoid any issues as these properties used to exist but don't exist anymore (even the MDN documentation for them have been removed).